pkgs/tools/system/swaysettings/default.nix

```diff
@@ -0,0 +1,75 @@
+{
+  pkgs,
+  meson,
+  vala,
+  pkg-config,
+  gettext,
+  ninja,
+  python3Full,
+  wrapGAppsHook,
+  appstream-glib,
+  desktop-file-utils,
+  gsettings-desktop-schemas,
+  glib,
+  gtk3,
+  libhandy,
+  json-glib,
+  libgee,
+  pantheon,
+  libxml2,
+  accountsservice,
+  gobject-introspection,
+  dbus
+}:
+with pkgs;
+pkgs.stdenv.mkDerivation rec {
+  pname = "swaysettings";
+  version = "0.0";
```

Comment:
If the software you are bringing to Nixpkgs has no formal release, use something like `version = "unstable-YYYY-MM-DD";`

Reply:
Corrected with last version